code/game/machinery/constructable_frame.dm

```diff
@@ -234,8 +235,19 @@ to destroy them and players will be able to make replacements.
 				user << "<span class='warning'>A fatal error with the board type occurred. Report this message.</span>"
 		else
 			user << "<span class='warning'>The multitool flashes red briefly.</span>"
-	else if(issolder(O))
-		local_fuses.Interact(user)
+	else */if(issolder(O))
+		//local_fuses.Interact(user)
+		var/t = input(user, "Which board should be designed?") as null|anything in allowed_boards
+		if(!t) return
+		var/obj/item/weapon/solder/S = O
+		if(!S.remove_fuel(4,user)) return
+		playsound(loc, 'sound/items/Welder.ogg', 100, 1)
+		if(do_after(user, src,40))
+			playsound(loc, 'sound/items/Welder.ogg', 100, 1)
+			var/boardType = allowed_boards[t]

```

Comment:
How do you figure?

Attack the blank board with a soldering iron twice and complete the input windows quickly enough and you either cause a runtime or dupe a completed board.